sdk/keyvault/azsecrets/internal/constants.go

```diff
@@ -9,8 +9,8 @@
 package internal
 
 const (
-	module  = "internal"
-	version = "v0.3.1"
+	moduleName    = "internal"
+	moduleVersion = "v0.2.0"
```

Comment:
Wrong version (sorry I think I forgot to update this).

Reply:
Also, since the constant name has changed, do we need to update the scrip that bumps the version automatically?

Reply:
PR for this: https://github.com/Azure/azure-sdk-for-go/pull/16801